{"name": "test_login_invalid_credentials", "status": "passed", "description": "Тест неуспешного входа с неверными логином и паролем.", "start": 1730469070955, "stop": 1730469071571, "uuid": "d9184f91-43f3-467e-8a99-e4f2db04cfe5", "historyId": "f28af0ae77ecccdafb01c7f80dfb30fd", "testCaseId": "f28af0ae77ecccdafb01c7f80dfb30fd", "fullName": "tests.test_user_login.TestLoginUser#test_login_invalid_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "MacBook-Pro-Daniil.local"}, {"name": "thread", "value": "8281-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}